qt/applications/workbench/workbench/plotting/figuremanager.py

```diff
@@ -333,6 +335,7 @@ def destroy(self, *args):
             # It seems that when the python session is killed,
             # Gcf can get destroyed before the Gcf.destroy
             # line is run, leading to a useless AttributeError.
+        self.deleteLater()
```

Comment:
Could I ask why this was necessary? Was the figuremanager not being deleted?

Reply:
Thanks for spotting these, I think this is left over from when I was trying a few different things. In any case it's not needed as this is in the destroy method which is called when the object is about to be deleted anyway!